src/client.ml

```diff
@@ -130,19 +124,37 @@ let convert pre =
   Dom.appendChild pre code;
   pre##.className := Js.string "language-ocaml"
 
+let remove_error_message n =
+  let p = Js.Unsafe.coerce n in
+  if (Js.string p##.firstChild == Js.string "[object Text]" && p##.firstChild##.data == Js.string "(* Error while translating to Reason *) \n") then (
+    Js.Opt.iter n##.firstChild (fun c -> ignore (n##removeChild c)) (*clean this*)
+  )
```

Comment:
Give a name to the string to make it easier to change the text without breaking everything.